docs/setup/main_setup.md

```diff
@@ -29,7 +29,7 @@ Use your distribution's package manager to install the following dependencies:
     1. Install dependencies
 
         ```sh
-        sudo apt install git git-lfs build-essential libssl-dev pkg-config libclang-dev rsync cmake libhdf5-dev python
+        sudo apt install git git-lfs build-essential libssl-dev pkg-config libclang-dev rsync cmake libhdf5-dev python3 glibc-source
```

Comment:
I don't think installing this package did anything against the problem you were facing. I just tried the whole setup process in an ubuntu docker container and it worked without installing it.
Please remove but keep the python package name change.

Reply:
Done